tests/__init__.py

```diff
@@ -320,7 +323,12 @@ def run_python(path, env=None, args=None):
         stdin=subprocess.PIPE,
         stdout=subprocess.PIPE,
     )
-    output, _ = p.communicate()
+    try:
+        output, _ = p.communicate(timeout=10)
+    except subprocess.TimeoutExpired:
+        p.kill()
+        output, _ = p.communicate(timeout=10)
+        return "%s\nFAIL - timed out" % output
```

Comment:
Please, avoid `%` interpolation when possible. Use `str.format`, e.g. `'{0}\n\nfail - timeout'.format(output)`. Besides new features it also doesn't have string/tuple ambiguity. `%` single argument must be enclosed in tuple when it comes from external code.